back/src/Model/Group.ts

```diff
@@ -113,9 +125,11 @@ export class Group implements Movable {
      */
     destroy(): void
     {
+        if (this.hasEmitedEvent) clientEventsEmitter.emitGroupDestroyed(this.roomId);
```

Comment:
Here, we should delete the timeout of 5 seconds (otherwise, the gauge will always be increased after 5 seconds!